config/settings.yml

```diff
@@ -210,6 +210,9 @@ claims_api:
         api_key: <%= ENV['claims_api__vanotify__services__lighthouse__api_key'] %>
         notification_client_secret: <%= ENV['claims_api__vanotify__services__lighthouse__notification_client_secret'] %>
         notify_service_id: <%= ENV['claims_api__vanotify__services__lighthouse__notify_service_id'] %>
+claims_evidence_api:
+  jwt_secret: <%= ENV['claims_evidence_api__jwt_secret'] %>
+  base_url: <%= ENV['claims_evidence_api__base_url'] %>
```

Comment:
These need to be in alphabetical order (in all three config files). Error in [settings-checks](https://github.com/department-of-veterans-affairs/vets-api/actions/runs/15172280311/job/42722601065?pr=22335):
```
Keys in config/settings.yml are not in alphabetical order:
  claims_evidence_api.["jwt_secret", "base_url"]
Keys in config/settings/development.yml are not in alphabetical order:
  claims_evidence_api.["jwt_secret", "base_url"]
Keys in config/settings/test.yml are not in alphabetical order:
  claims_evidence_api.["jwt_secret", "base_url"]
```